content/en/developers/integrations/agent_integration.md

```diff
@@ -440,6 +440,33 @@ sudo datadog-agent integration install -w /path/to/wheel.whl
 
 Once you have created your Agent-based integration, see the [Create a tile][20] documentation for information on populating the remaining [required assets][31] that appear on your integration tile, and opening a pull request.
 
+## Update your integration
+To update your integration, edit the relevant files and open a new pull request to your integration's directory in the [`integrations-extras`][21] or [`marketplace`][22] repository. 
+
+* If you are editing or adding new integration code, a version bump is required.
+
+* If you are editing or adding new README content, manifest information, or assets such as dashboards and recommended monitors, a version bump is optional. 
+
+With or without a version bump, updates to assets such as dashboards and recommended monitors are available to customers after the corresponding pull requests are merged and the assets are published. Updates to `README.md`, `manifest.json`, or any other non-code files are also immediately available to customers after publishing. 
```

Comment:
Is this true? I know that our team syncs assets once per week. Naturally if something is urgent, that gets synced separately, but I'm not sure that's the same expectation as what this sentence sets up.

Reply:
@nmuesch knows better than me here

Reply:
my two cents would be to make point 2 more explicit. You can literally say something along the lines of your comment: "no more action needed from you [the contributor], we take care of it".

As for point 1: our team syncs assets individually for cases where we know there are changes. For instance releases of new integrations or urgent fixes to assets that we make ourselves and that somebody's waiting for. The weekly sync captures non-urgent changes and/or changes by external contributors to core and extras repos.

Reply:
@iliakur @nmuesch made some updates based on this comment thread. Let me know what you think

Reply:
lgtm

Reply:
Agreed, the updates look good to me too!

Reply:
Great! @alai97 can we merge plz?